Binance.Net/BinanceClient.cs

```diff
@@ -1415,10 +1437,341 @@ public async Task<WebCallResult<object>> StopUserStreamAsync(string listenKey)
+
+            return await ExecuteRequest<BinanceCanceledOrder>(GetUrl(CancelMarginOrderEndpoint, MarginApi, MarginVersion), DeleteMethod, parameters, true).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Query loan records
+        /// </summary>
+        /// <param name="asset">The records asset</param>
+        /// <param name="transationId">The id of loan transation</param>
+        /// <param name="startTime">Time to start getting records from</param>
+        /// <param name="endTime">Time to stop getting records to</param>
+        /// <param name="current">Number of page records</param>
+        /// <param name="size">The records count size need show</param>
+        /// <param name="receiveWindow">The receive window for which this request is active. When the request takes longer than this to complete the server will reject the request</param>
+        /// <returns>Loan records</returns>
+        public WebCallResult<BinanceQueryLoan[]> QueryLoan(string asset, long? transationId = null, DateTime? startTime = null, DateTime? endTime = null, int? current = 1, int? size = 10, long? receiveWindow = null) => QueryLoanAsync(asset, transationId, startTime, endTime, current, size, receiveWindow).Result;
+
+        /// <summary>
+        /// Query loan records
+        /// </summary>
+        /// <param name="asset">The records asset</param>
+        /// <param name="transationId">The id of loan transation</param>
+        /// <param name="startTime">Time to start getting records from</param>
+        /// <param name="endTime">Time to stop getting records to</param>
+        /// <param name="current">Number of page records</param>
+        /// <param name="size">The records count size need show</param>
+        /// <param name="receiveWindow">The receive window for which this request is active. When the request takes longer than this to complete the server will reject the request</param>
+        /// <returns>Loan records</returns>
+        public async Task<WebCallResult<BinanceQueryLoan[]>> QueryLoanAsync(string asset, long? transationId = null, DateTime? startTime = null, DateTime? endTime = null, int? current = 1, int? size = 10, long? receiveWindow = null)
+        {
+            var timestampResult = await CheckAutoTimestamp().ConfigureAwait(false);
+            if (!timestampResult.Success)
+                return new WebCallResult<BinanceQueryLoan[]>(timestampResult.ResponseStatusCode, timestampResult.ResponseHeaders, null, timestampResult.Error);
+
+            var parameters = new Dictionary<string, object>
+            {
+                { "asset", asset },
+                { "timestamp", GetTimestamp() }
+            };
+            parameters.AddOptionalParameter("txId", transationId?.ToString());
+            // TxId or [startTime, endTime) must be sent. txId takes precedence.
```

Comment:
Add a check to see if at least 1 is provided

Reply:
symbol | STRING | YES |  
orderId | LONG | NO |  
origClientOrderId | STRING | NO |  
newClientOrderId | STRING | NO | Used to uniquely identify this cancel. Automatically generated by default.
recvWindow | LONG | NO |  
timestamp | LONG | YES